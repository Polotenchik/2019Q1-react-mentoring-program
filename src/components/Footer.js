import React from 'react';
import { Logo }  from './Logo';

export const Footer = () => (
    <footer className={'footer'}>
        <Logo />
    </footer>
);