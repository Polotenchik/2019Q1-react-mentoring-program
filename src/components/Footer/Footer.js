import React from 'react';
import styled from 'styled-components';
import { Logo } from '../index';

const ContainerFooter = styled.footer`
    height: 60px;
    width: 1150px;
    background: #434343;
    flex: 0 0 auto;
    margin: 0 auto;

    img {
        margin: 20px auto auto 50px; 
    }
`;

const Footer = () => (
     <ContainerFooter>
         <Logo />
     </ContainerFooter>
);

<<<<<<< HEAD

// const Footer = () => (
//     <footer className={'footer'}>
//         <Logo />
//     </footer>
// );

=======
>>>>>>> master
export default Footer;
