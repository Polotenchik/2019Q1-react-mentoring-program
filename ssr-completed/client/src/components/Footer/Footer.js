import React from 'react';
import styled from 'styled-components';
import Theme from '../../common/Theme/Theme';
import Logo from '../../common/Components/Logo/Logo';

const Footer = () => (
  <FooterBackground>
    <FooterWrapper>
      <Logo />
    </FooterWrapper>
  </FooterBackground>
);

export default Footer;

const FooterBackground = styled.footer`
    flex: 0 0 auto;    
    width: 100%;
    background-color: ${Theme.colors.lighter_grey};
`;

const FooterWrapper = styled.div`
    width: 80%;
    margin: 0 auto;
    padding: 10px 0;
`;
