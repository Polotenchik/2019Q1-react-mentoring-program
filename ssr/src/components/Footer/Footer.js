import React from 'react';
import { Logo }  from '../index';

export const Footer = () => (
    <footer className={'footer'}>
        <Logo />
    </footer>
);