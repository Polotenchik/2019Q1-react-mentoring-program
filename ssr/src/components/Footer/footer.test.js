import React from 'react';
import { Footer } from './Footer';
import { shallow } from 'enzyme';

describe('<Footer />', () => {
  describe('<Footer />', () => {
    it('should render footer', () => {
      const wrapper = shallow(<Footer />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});