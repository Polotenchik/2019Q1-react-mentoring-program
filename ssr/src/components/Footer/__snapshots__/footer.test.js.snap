// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> <Footer /> should render footer 1`] = `
<footer
  className="footer"
>
  <Logo />
</footer>
`;
